diff --git a/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh b/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh
index fbb4094..c9824d9 100755
--- a/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh
+++ b/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh
@@ -213,7 +213,6 @@ build_slice() {
     -destination "generic/platform=${destination}" \
     -derivedDataPath "$DERIVED_DATA_PATH" \
     -configuration "$CONFIGURATION" \
-    -quiet \
     -disableAutomaticPackageResolution \
     -skipPackagePluginValidation \
     -skipMacroValidation \
